website/project/model.py

```diff
@@ -1896,6 +1906,10 @@ def root(self):
             return self
 
     @property
+    def registrations(self):
+        return self.node__registrations.find(Q('archiving', 'eq', False))
```

Comment:
There may be a slightly more efficient way to do this, do you know if `MongoQuery`s do anything upon being created?